Content.Server/Clothing/MaskSystem.cs

```diff
@@ -98,6 +101,9 @@ private void ToggleMaskComponents(EntityUid uid, MaskComponent mask, EntityUid w
             if (isEquip || !TryComp<BreathToolComponent>(uid, out var breathTool))
                 return;
 
+            if (mask.MaskShoutAction != null)
+                mask.MaskShoutAction.Enabled = !mask.IsToggled;
```

Comment:
having the mask toggled to shout doesn't make sense?

Reply:
Toggled means down. It confused me too. Have you tested this code by any chance?

Reply:
To clarify, you have to have the mask on in order for the shout action to be enabled. This is consistent with the behavior on SS13.